internal/node-patches/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@build_bazel_rules_nodejs//:tools/defaults.bzl", "npm_package")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
```

Comment:
where is the build step to transform .ts to .js?

Reply:
npm run compile

Reply:
the question you asked indirectly with this that i didn't answer is where is the build rule that ties the development of this package into the workflow of the monorepo.

i need test rule that runs npm test. in the npm workflow this causes compile to be run. It's up to you how much of package json scripts we want to re-implement in rules but less is better imo.

Reply:
seems to me that the package.json test script should invoke bazel rather than the other way around. let's discuss tomorrow?

Reply:
we should ensure that the tests you add in this PR are getting run on CI.

I'm okay with this package being "different" in the sense that it's bazel layered on top of npm scripts instead of the other way around. let's figure out how to run the `npm test` command under bazel